administrator/components/com_media/resources/scripts/mediamanager.js

```diff
@@ -20,6 +21,7 @@ import store from './store/store';
 Vue.use(Translate);
 
 // Register the vue components
+Vue.component('media-adapter', Adapter);
```

Comment:
I prefer disk and drivers.

Driver is a representation of a media provider which can have multiple disks and internally uses the Adapter to talk to the provider.